### PR TITLE
Update Component Pages and Readme

### PR DESCRIPTION
**Pull request by varun13169**

### CODE REVIEW

**Review by zoef-yt — APPROVED** [review]
Nice addition of gif in readme looks great. Good to merge.

File: README.md [hunk 1]
```diff
@@ -1,24 +1,49 @@
-# </ Delta UI >
+<h1 align="center">&lt;/ Delta UI &gt;</h1>
+
+[![Generic badge](https://img.shields.io/badge/USES-HTML-green.svg)](https://shields.io/) [![Generic badge](https://img.shields.io/badge/USES-CSS-green.svg)](https://shields.io/) [![Generic badge](https://img.shields.io/badge/USES-JS-green.svg)](https://shields.io/)
 
 ## Installation
 
 Paste the following CDN link as part of the impot in you main CSS and you are good to go.
```
Comment (Muzammil-khan710) [inline README.md]: Typo

**Review by Muzammil-khan710 — APPROVED** [review]
Readme is really great
Good to merge !

File: README.md (hunk 1 above)
Comment (varun13169) [reply README.md]: Done.

**Pull request merged**